ddl/table.go

```diff
@@ -586,7 +586,7 @@ func verifyNoOverflowShardBits(s *sessionPool, tbl table.Table, shardRowIDBits u
 	if err != nil {
 		return errors.Trace(err)
 	}
-	if tables.OverflowShardBits(autoIncID, shardRowIDBits, autoid.RowIDBitLength) {
+	if tables.OverflowShardBits(autoIncID, shardRowIDBits, autoid.RowIDBitLength, true) {
```

Comment:
should judge the column signed character in the upper function caller and pass the parameter？

Reply:
`shard_rowid_bits` always reserve the sign bit.